uwac/libuwac/uwac-window.c

```diff
@@ -121,7 +121,7 @@ static void xdg_handle_toplevel_configure(void* data, struct xdg_toplevel* xdg_t
 	event->window = window;
 	event->states = surfaceState;
 
-	if (width && height)
```

Comment:
Shouldn´t this just check if the `width` and `height` actually changed?

Reply:
I think this should check if the width and height is valid. But it is maybe an improvement to check (window->width != width || window->height != height) also. 

but the problem still exists that the request was not sucessful, the window size hangs on the state before and the compositor is not informed about it.